multicluster/controllers/multicluster/common/helper.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 Antrea Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "k8s.io/apimachinery/pkg/types"
+
+const (
+	AntreaMcsLabel        = "antrea.io/multi-cluster"
```

Comment:
the main reason is there is no annotation selector, so I use label.

Reply:
Sorry, how we use label selectors?

Reply:
we use labelselector to filter resource export in resourceexport_controller.go, eg  https://github.com/antrea-io/antrea/pull/3024/files#diff-4a5af6131176d9b4bd232b0185161872343c0f501be3f061e07a555cc3b2dcb8R168
and 
https://github.com/antrea-io/antrea/pull/3024/files#diff-4a5af6131176d9b4bd232b0185161872343c0f501be3f061e07a555cc3b2dcb8R318 etc. I feel in some cases we can use annotation, and some other cases we need label so we can use label selector. for service controller part, I will check if possible to use annotation instead.

Reply:
I feel ideally we should not add labels in user created resources. It is redundant data, and more importantly we cannot prevent users from removing the labels. If example, they can just apply a yaml to update an existing Services and that will remove labels added by us.

Reply:
indeed, that make sense, thanks to point out, I will check how to achieve the same feature in a different way.

Reply:
hi @jianjuns  I updated it with an annotation instead of a label, the main purpose of this info is to reduce the service/endpoint update events and watch those exported resource event only https://github.com/antrea-io/antrea/pull/3024/commits/71c418fb9cb50da4f0a5c7268fe0f2cd0de7192c#diff-1e1d6e2558b5bb86a952c580e877a8a072c499a0c278194afb4d0948ae459d10R407-R443, we can skip add ingany annotation and watch all services/endpoints, but I feel it will be a big effort to handle unnecessary services/endpoints change in ServiceExport reconciler. let me know what's your thought. thanks.

Reply:
But annotation is the same as label that can be removed by users (sorry if my previous comments are not clear). I feel watching all Services and Endpoints is ok, as it is just by one controller in the cluster. AntreaProxy watches all Services and Endpoints in all hosts.